src/emu/http.cpp

```diff
@@ -16,6 +16,8 @@ HTTP server handling
 #include <inttypes.h>
 #include <stdint.h>
 
+#undef DEBUG
+#define DEBUG 0
```

Comment:
Use the `osd_printf_*` functions rather than rolling your own mechanism like this.

Reply:
Done!